src/platform/NodePlatformFunctions.ts

```diff
@@ -119,6 +125,17 @@ class NodePlatformFunctions extends PlatformFunctions {
         });
     });
   }
+
+  /** @override */
+  createNodeHttpClient(agent: http.Agent): typeof HttpClient {
+    // @ts-ignore
+    return new NodeHttpClient(agent);
+  }
+
+  /** @override */
+  createNodeCryptoProvider(): StripeCryptoProvider {
```

Comment:
nit: Isn't StripeCryptoProvider a class can we reference it the same way we do HttpClient?

Reply:
`StripeCryptoProvider` is the type defined in https://github.com/stripe/stripe-node/blob/5c444308d744bb806b50f915abd1255c0bd17434/src/Types.d.ts#L91, but after seeing this I want to try defining a `StripeHttpClient` type so we don't have to import so many things

Reply:
We should do the opposite, use as much of existing types as possible. The goal is to remove Types.d.ts it's a temporary hack.

Reply:
Gotcha, sounds good!

Reply:
Replaced with `CryptoProvider` in https://github.com/stripe/stripe-node/pull/1679/commits/b43f1fb28dac6ae470e08feedc998e1aafbe1eee